package.json

```diff
@@ -14,7 +14,12 @@
     "prebuild": "npm run localize",
     "pretest": "npm run localize",
     "lint:style": "d2-style js check --all",
-    "cypress:open": "cypress open"
+    "cypress:start": "BROWSER=none yarn start",
+    "cypress:envtest": "node ./scripts/check_env_vars.js",
+    "cypress:browser": "cypress open --config baseUrl=http://localhost:3000,defaultCommandTimeout=15000",
+    "cypress:ci": "NO_COLOR=1 cypress run -b $BROWSER --config baseUrl=http://localhost:3000,defaultCommandTimeout=15000",
+    "cypress:open": "yarn cypress:envtest && concurrently --kill-others -n cra,cypress 'yarn cypress:start' 'wait-on http-get://localhost:3000 && yarn cypress:browser --env LOGIN_URL=$REACT_APP_DHIS2_BASE_URL'",
```

Comment:
What does the `-n cra,cypress` switch do and what does the names map to?

Reply:
it's for prefixing the output of concurrently. The first command (`'yarn cypress:start'`) will be prefixed with "cra" and the second one (`'wait-on http-get://localhost:3000 && yarn cypress:browser --env LOGIN_URL=$REACT_APP_DHIS2_BASE_URL'`) with "cypress"

Reply:
I chose `cra` because the import export app is a create-react-app. Maybe `yarn start` is better?

Reply:
I think it's fine. I didn't know if it was bound to the scripts in `package.json` or something.